client/components/LearnerViewDirective.js

```diff
@@ -35,8 +35,10 @@ tie.directive('learnerView', [function() {
                   </div>
                   <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
                 </div>
+                <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
               </div>
             </div>
+          </div>
```

Comment:
But when I sum up the div tags, they match up numerically, but I think something is not right given this lone &lt;/div&gt; tag was added here. Hence me request for you to double-check the tags to make sure they match.